code/datums/components/aiming.dm

```diff
@@ -6,7 +6,7 @@
 #define DROP_WEAPON "drop_weapon"
 #define DROP_TO_FLOOR "drop_to_floor"
 #define CANCEL "cancel"
-#define FIRE "fire"
+#define SHOOT "fire"
```

Comment:
Gun aiming component I had to rename

Reply:
might as well change the string too while you are at it

Reply:
gotcha